Headers/AppKit/NSLayoutAnchor.h

```diff
@@ -0,0 +1,97 @@
+/* Interface of class NSLayoutAnchor
+   Copyright (C) 2020 Free Software Foundation, Inc.
+   
+   By: Gregory Casamento <greg.casamento@gmail.com>
+   Date: Sat May  9 16:30:52 EDT 2020
```

Comment:
This is fine, but just a check, do we want to put the full timestamp here? I think it was usually just month+year. Don't mind this either.

Reply:
I'll change the script to shorten it.